### PR TITLE
RPR 2.2.7  / Rif 1.7.1

### PR DESCRIPTION
**Pull request by Vsevolod1983**
### PURPOSE
Let's make some update regarding RPR and RIF

### CODE REVIEW

File: cmake/modules/FindRif.cmake [hunk 1]
```diff
@@ -35,8 +35,11 @@ find_library(RIF_LIBRARY
 if(WIN32)
     set(RIF_BINARIES
         ${RIF_LOCATION_LIB}/dxcompiler.dll
+	${RIF_LOCATION_LIB}/dxil.dll
         ${RIF_LOCATION_LIB}/MIOpen.dll
         ${RIF_LOCATION_LIB}/RadeonImageFilters.dll
+        ${RIF_LOCATION_LIB}/OpenImageDenoise.dll
```
Comment (hshakula) [inline cmake/modules/FindRif.cmake:41]: Why do we need it?

**Review by hshakula — CHANGES_REQUESTED** [review]

File: cmake/modules/FindRif.cmake (hunk 1 above)
Comment (Vsevolod1983) [reply cmake/modules/FindRif.cmake:41]: I added everything RIF repo has

Comment (Vsevolod1983) [reply cmake/modules/FindRif.cmake:41]: At least RadeonML.dll is needed for sure. dxil.dll probably same

Comment (hshakula) [reply cmake/modules/FindRif.cmake:41]: Please remove this line, we don't use OIDN.

Comment (bsavery) [reply cmake/modules/FindRif.cmake:41]: What about for CPU + denoising?  Or at least we should discuss for future if that makes sense for final renders

Comment (hshakula) [reply cmake/modules/FindRif.cmake:41]: I only said that we don't use it at the moment so it makes no sense to add this 40 Mb dll to the package.

Surely we can use the OIDN if needed. It's more robust than the AI denoiser. Do you want me to add a way for users to select a denoiser filter and add support of the OIDN?

Comment (bsavery) [reply cmake/modules/FindRif.cmake:41]: What do we currently if CPU is enabled and denoising is turned on?  The RIF denoiser uses OIDN on cpu

Comment (hshakula) [reply cmake/modules/FindRif.cmake:41]: https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderUSD/blob/master/pxr/imaging/plugin/hdRpr/rprApi.cpp#L2163

EAW denoiser is used in this case.

> The RIF denoiser uses OIDN on cpu

This is really surprising if it's true.

Comment (bsavery) [reply cmake/modules/FindRif.cmake:41]: Sorry @hshakula I should amend my comment "RIF AI Denoise" uses OIDN if run on CPU.

Comment (hshakula) [reply cmake/modules/FindRif.cmake:41]: What do you guys mean when you say RIF running on CPU? Do you able to create OpenCL context on CPU? For me, RIF never detected my CPU as a valid device.

Comment (bsavery) [reply cmake/modules/FindRif.cmake:41]: Let's keep it as is for now (no OIDN) and talk about this in a future task.

**Review by hshakula — APPROVED** [review]

**Pull request merged**